### ISSUE TITLE
BUG: Prod user cannot create second VC

### ISSUE THREAD

**Issue by Comoque1** [alkem-io/client-web#7143]
**Describe the bug**
A production user cannot create a VC, considering, that she is:
- userID: "9e7e3cdc-a5b2-4c70-94df-488996b0470c"
- Roles: 
  - Beta tester
  - VC campaign
  - admin on both of the spaces under her account
- already has 1 created vc, that has no body of knowledge
- policy reset on account didn't help
- server logs show [warnings](https://kibana.alkem.io/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:60000),time:(from:now-30m,to:now))&_a=(columns:!('@timestamp',context,userId,message,stack,level),filters:!(),index:ee214087-bb19-4337-b7ec-a593912a6afd,interval:auto,query:(language:kuery,query:'9e7e3cdc-a5b2-4c70-94df-488996b0470c'),sort:!(!('@timestamp',desc))))
<img width="1096" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNVdZQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f1371e3185ca0333b789800cf768b993a75172ed/image.png" alt="image.png" />
<img width="1363" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWFZQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6b8408e7ffc26615e198caacf6fc14ea81ff5ecd/image.png" alt="image.png" />

**To Reproduce**

**Expected behavior**
VC addition to a space to be a separate step. After the VC is created, the user should be asked if they want to add the VC to one of their spaces - and then show the list of their spaces with the spaces where VC usage is not enabled to be shown but grayed out.

**Comment by bobbykolev** [comment]
This dialog is presented if the user lacks one of the following privileges (on the Space):
CREATE_SUBSPACE,
ACCESS_VIRTUAL_CONTRIBUTOR,
COMMUNITY_ADD_MEMBER_VC_FROM_ACCOUNT

Here's a PR in which the client starts to send **Sentry** Info log in case of such insufficient privileges - https://github.com/alkem-io/client-web/pull/7144

**Comment by techsmyth** [comment]
So the finger points very strongly to the space not being entitled for virutal contributors. And checking her spaces: <img width="686" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK09iQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--021c9ae0047b300c53c055aa56bc45f99e25d2d0/image.png" alt="image.png" />

It is clear that the first space does not have VCs enabled - which is presumably what the client is checking. 

Quick fix is to add the license plan for VCs to the second space. Clearly better feedback would help, and we need to reconsider the adding of VCs to a space. @SimoneZaza @larssondenise  fyi

**Comment by SimoneZaza** [comment]
@Comoque1 I assume this will be fixed with all the changes? Moving to QA

**Comment by Comoque1** [comment]
@SimoneZaza, correct, with the current implementation, it should not be the case, as long as the user doesn't hit the limit.
